Assets/PatchKit Patcher/Scripts/AppData/Local/ZipUnarchiver.cs

```diff
@@ -69,7 +69,8 @@ private void UnarchiveEntry(ZipEntry zipEntry)
 
         protected virtual void OnUnarchiveProgressChanged(string name, bool isFile, int entry, int amount, double entryProgress)
         {
-            if (UnarchiveProgressChanged != null) UnarchiveProgressChanged(name, isFile, entry, amount, entryProgress);
+            var handler = UnarchiveProgressChanged;
+            if (handler != null) handler(name, isFile, entry, amount, entryProgress);
```

Comment:
Don't use if statements without curly brackets. Please change it to:
```
if (...)
{
    // body
}
```

Reply:
Done.